app/scripts/platforms/extension.js

```diff
@@ -26,6 +26,14 @@ class ExtensionPlatform {
       cb(e)
     }
   }
+
+  isOnline () {
+    return navigator.onLine
+  }
+
+  addOnLineListener (cb) {
+    window.addEventListener('online', cb)
```

Comment:
Yeah camel cased correctly here. Incorrect capitalization kinda driving me nuts, going to change it myself.